packages/tooltip/Tooltip.test.tsx

```diff
@@ -1,17 +1,49 @@
-import { Tooltip } from './Tooltip';
-
+import { Tooltip } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Tooltip.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Tooltip!');
-
     return mount(
       <Tooltip label="hello"/>
-    ).wait();
+    ).wait((element) => {
+      expect( element.has('.c-tooltip') ).to.be.true;
+      expect( element.has('.c-tooltip--right') ).to.be.true;
+      expect( element.one('span').node.getAttribute('aria-label') ).to.equals( 'hello' );
+    });
+
+  });
+
+  it( 'should render element', () => {
+
+    return mount(
+      <bl-tooltip label="hello"></bl-tooltip>
+    ).wait((element) => {
+      expect( element.has('.c-tooltip') ).to.be.true;
+      expect( element.has('.c-tooltip--right') ).to.be.true;
+      expect( element.one('span').node.getAttribute('aria-label') ).to.equals( 'hello' );
+    });
+
+  });
+
+  it( 'should render with top tooltip', () => {
```

Comment:
Good idea with the two `describe` - I was thinking how we will want to structure this.